lib/pbench/server/database/__init__.py

```diff
@@ -4,7 +4,7 @@
 """
 from pbench.server.database.models.active_tokens import ActiveTokens  # noqa F401
 from pbench.server.database.models.users import User  # noqa F401
-from pbench.server.database.models.tracker import Dataset, Metadata  # noqa F401
+from pbench.server.database.models.datasets import Dataset, Metadata  # noqa F401
```

Comment:
I can't imagine why this would have changed. Then again, I believe it was only reported by `LGTM` and not by `flake8` so who knows? I'm not convinced it's worth the experiment for this PR, however.

Reply:
I'm generally opposed to having those sorts of directives in the code (but, I understand that, at times, they are a necessary evil), and I figured that with all the various changes to related code they might not be required anymore, so I thought the question was worth asking.  But, if you think that they are still needed, then let's move along.

Reply:
We need to have all the models imported in order to properly initialize the database, but there's no appropriate module in which to implement that initialization which actually *uses* the full set of model imports, so kindly LGTM will inevitably become grumpy and unpleasant unless we at least mess up the code a bit with ugly markers to appease it.